contrib/cloudflare.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Cloudflare DNS Updater
+Documentation=https://github.com/favonia/cloudflare-ddns/blob/main/README.markdown
+Wants=network-online.target
```

Comment:
Should this be `After=`? I read the documentation and it seems this would allow the tool to run even if the network is not up. Is this the desired result?